mastracode/src/agents/tools.ts

```
@@ -14,6 +14,7 @@ import type { ToolHooks } from '@mastra/core/tools';
 import type { HookManager } from '../hooks';
 import type { McpManager } from '../mcp';
 import type { MastraCodeComposedState } from '../schema';
+import { readHarnessState } from '../utils/harness-state.js';
```

ESLint

[error] `../utils/harness-state.js` import should occur after import of `../tools`

(import/order)